### PR TITLE
(consul & google) report if server group is disabled

### PR DESCRIPTION
**Pull request by lwander**
Whenever consul is running for any instance in a server group, we start taking consul health into account.

ycafp5aqzvc

### CODE REVIEW

**Comment by lwander** [comment]
@duftler

**Comment by lwander** [comment]
oh, and @jtk54 PTAL since this touches some of your L7 logic

**Comment by jtk54** [comment]
LGTM

File: clouddriver-consul/src/main/groovy/com/netflix/spinnaker/clouddriver/consul/model/ConsulNode.groovy [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2016 Netflix, Inc.
```
Comment (duftler) [inline clouddriver-consul/src/main/groovy/com/netflix/spinnaker/clouddriver/consul/model/ConsulNode.groovy]: s/Netflix/Google

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/view/GoogleClusterProvider.groovy [hunk 2]
```diff
@@ -180,21 +181,43 @@ class GoogleClusterProvider implements ClusterProvider<GoogleCluster.View> {
...
       // We have a mix of L4 and L7.
-      serverGroup.disabled = serverGroup.disabled && httpDisabledStates.every { it }
+      serverGroup.disabled &= httpDisabledStates.every { it }
+    }
+
+    // Now that disabled is set based on L7 & L4 state, we need to take Consul into account.
+    if (consulRunning) {
+      // If there are no load balancers to determine enable/disabled status we rely on Consul exclusively.
+      if (loadBalancers.size() == 0) {
+          serverGroup.disabled = true
```
Comment (duftler) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/view/GoogleClusterProvider.groovy]: Indentation is off here.

**Pull request merged**